src/main/java/com/google/api/codegen/transformer/DynamicLangApiMethodTransformer.java

```diff
@@ -71,6 +71,9 @@ public OptionalArrayMethodView generateMethod(
       apiMethod.pageStreamingView(
           pageStreamingTransformer.generateDescriptor(
               context.getSurfaceInterfaceContext(), method));
+    } else if (context.getMethodConfig().isLongRunningOperation()) {
+      apiMethod.longRunningView(lroTransformer.generateDetailView(context));
+      apiMethod.type(ClientMethodType.OperationOptionalArrayMethod);
```

Comment:
One thing I was chatting to Michael B about, and which is independent of this but may diminish confusion:

At the moment, some of  the client method type enum values seem to be used in the same way (AFAICT) in different languages even though they we use distinct values. It might be desirable to use the same values across languages when they have the same semantics.

This would be analogous to, but independent of, using the same calling form enum value (once we create the enum type) across different languages, where applicable.